Source/Languages/Casing/CaseStyle.ts

```diff
@@ -11,14 +11,29 @@ namespace GLS.Languages.Casing {
         None,
 
         /**
-         * Pascal case, such as MyVariable.
+         * Camel case, such as myVariable.
          */
-        PascalCase,
+        CamelCase,
 
         /**
-         * Camel case, such as myVariable.
+         * File system case, such as "my/variable.
          */
-        CamelCase,
+        FileSystem,
+
+        /**
+         * Packages in lower case, such as "my.variable".
+         */
+        PackageLowerCase,
+
+        /**
+         * Packages in upper case, such as "My.Variable".
+         */
+        PackageUpperCase,
```

Comment:
Should we call this something unrelated to code? 
Like "PeriodSeparatedUpperCase" or something like that. There could be other things that use this format besides packages.